docs/feature-support-matrix.md

```diff
@@ -40,12 +44,12 @@ graph-node: >=0.35.0 <0.36.0
 
 ### Latest Council snapshot
 
-[GGP-0040: Updated Feature Support (ens, arweave, 0.35.x)](https://snapshot.org/#/council.graphprotocol.eth/proposal/0x214ad88959a8e09e7341b21650699ab67345493d9386118dc524c827bd011090)
+[GGP-0042: Updated Feature Matrix Support (base, scroll, linea)](https://snapshot.org/#/council.graphprotocol.eth/proposal/0x229e88da4839f8ba7f349323dfc564f991f72e4faef84506a7205b1c5059f631)
```

Comment:
Add bsc for completeness?

Reply:
This is outdated. Adding the latest GGP, which includes bsc.